src/libraries/Microsoft.Extensions.DependencyInjection.Specification.Tests/src/ActivatorUtilitiesTests.cs

```diff
@@ -329,7 +323,6 @@ public void GetServiceOrCreateInstanceRegisteredServiceSingleton()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/33894", TestRuntimes.Mono)]
```

Comment:
How are we now ensuring to skip on Mono after this change?

Reply:
From above:

> I've removed all the ActiveIssue attributes in order to get new data from CI to see if these are all still failing on mono or not. If so I will see see about adding attributes to the derived tests classes in DI.External.Tests.

Issue 33894 was created to track a very specific test that was tuned to the stack size on CoreCLR.  I want to find out what's failing and how and create a different issue, with potentially more granular active issues.

Reply:
So it looks like the ExtenralProvider tests only fail under the interpreter with segfault.
All the tests fail in WASM due to lack of events/threading.  I'll split these apart.